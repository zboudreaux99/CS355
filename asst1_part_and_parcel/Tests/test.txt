abc, def, ghi
"j\"kl","mn,o"