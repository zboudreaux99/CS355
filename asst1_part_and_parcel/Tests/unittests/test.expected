> Task :compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources NO-SOURCE
> Task :testClasses
> Task :test

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.5/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1s
3 actionable tasks: 2 executed, 1 up-to-date
